### PR TITLE
Added more Civilian Props

### PR DESCRIPTION
**Pull request by MustaphaTR**
Pipes, V3 Ammunition, Tires, Practice Target and Derelict Tank. V3 Ammo has different explosion like in orginal.
miscs

### CODE REVIEW

File: sequences/civilian-structures.yaml [hunk 1]
```diff
@@ -277,6 +277,60 @@ camisc04:
 		ShadowStart: 5
 		Offset: 0, -15
 
+camisc04:
```
Comment (Mailaender) [inline sequences/civilian-structures.yaml]: > Exception of type `System.ArgumentException`: MiniYaml.Merge, duplicate values found for the following keys: camisc04

Comment (MustaphaTR) [inline sequences/civilian-structures.yaml]: lol, it looks like i copied too much thing.

**Comment by MustaphaTR** [comment]
Can non squash right now. I'm not on computer, possibly can't open it today. Possibly can do it tomorrow.

**Comment by Phrohdoh** [comment]
When you take ingame screenshots (for PR purposes) when introducing new actors (any changing artwork, really) please turn on the terrain geometry overlay.

**Comment by MustaphaTR** [comment]
Squashed.

File: rules/civilian-props.yaml [hunk 2]
```diff
@@ -42,12 +42,74 @@ camisc04:
...
 	Health:
 		HP: 10
 	Armor:
 		Type: Concrete
 
+camisc05:
+	Inherits: ^CivBuilding
+	Building:
+		Footprint: x
+		Dimensions: 1, 1
```
Comment (abcdefg30) [inline rules/civilian-props.yaml]: This shouldn't be needed as `^CivBuilding` inherits `^Building` and this has [this](https://github.com/OpenRA/ra2/blob/9ef4a12c66258e95093bf3e651d6866382a9046d/rules/defaults.yaml#L77-L79) already defined.

(Same for `camsc11` and `camsc12`.)

**Comment by MustaphaTR** [comment]
Updated.

**Comment by Phrohdoh** [comment]
This all looks sensible to me, but I have not tested it ingame.

**Comment by Mailaender** [comment]
Thanks!

**Pull request merged**